__device-tests__/helpers/caps.js

```diff
@@ -1,7 +1,7 @@
 exports.ios12 = {
 	browserName: '',
 	platformName: 'iOS',
-	platformVersion: '12.2',
+	platformVersion: '12.4',
```

Comment:
Any specific reason for this change? This updates the version used on CI and I'm not sure if that's available as yet